### ISSUE TITLE
dotenv_macros fails to compile on nightly 1.11.0

### ISSUE THREAD

**Issue by pwoolcoc** [slapresta/rust-dotenv#43]
Here is the error I get when I try to compile `dotenv` on the latest nightly:

```
/home/paul/.cargo/registry/src/github.com-1ecc6299db9ec823/dotenv_codegen-0.8.1/src/dotenv_macro.rs:13:66: 13:80 error: type name `ast::TokenTree` is undefined or not in scope [E0412]
/home/paul/.cargo/registry/src/github.com-1ecc6299db9ec823/dotenv_codegen-0.8.1/src/dotenv_macro.rs:13 pub fn expand_dotenv<'cx>(cx: &'cx mut ExtCtxt, sp: Span, tts: &[ast::TokenTree])
                                                                                                                                                                        ^~~~~~~~~~~~~~
/home/paul/.cargo/registry/src/github.com-1ecc6299db9ec823/dotenv_codegen-0.8.1/src/dotenv_macro.rs:13:66: 13:80 help: run `rustc --explain E0412` to see a detailed explanation
/home/paul/.cargo/registry/src/github.com-1ecc6299db9ec823/dotenv_codegen-0.8.1/src/dotenv_macro.rs:13:66: 13:80 help: you can import it into scope: `use syntax::tokenstream::TokenTree;`.
/home/paul/.cargo/registry/src/github.com-1ecc6299db9ec823/dotenv_codegen-0.8.1/src/dotenv_macro.rs:25:50: 25:64 error: type name `ast::TokenTree` is undefined or not in scope [E0412]
/home/paul/.cargo/registry/src/github.com-1ecc6299db9ec823/dotenv_codegen-0.8.1/src/dotenv_macro.rs:25 fn expand_env(cx: &mut ExtCtxt, sp: Span, tts: &[ast::TokenTree])
                                                                                                                                                        ^~~~~~~~~~~~~~
/home/paul/.cargo/registry/src/github.com-1ecc6299db9ec823/dotenv_codegen-0.8.1/src/dotenv_macro.rs:25:50: 25:64 help: run `rustc --explain E0412` to see a detailed explanation
/home/paul/.cargo/registry/src/github.com-1ecc6299db9ec823/dotenv_codegen-0.8.1/src/dotenv_macro.rs:25:50: 25:64 help: you can import it into scope: `use syntax::tokenstream::TokenTree;`.
error: aborting due to 2 previous errors
Build failed, waiting for other jobs to finish...
error: Could not compile `dotenv_codegen`.

To learn more, run the command again with --verbose.
```

rustc version: `rustc 1.11.0-nightly (696b703b5 2016-07-03)`

**Comment by iamsebastian** [comment]
https://github.com/diesel-rs/diesel/issues/370

**Comment by sgrif** [comment]
Fixed in #44 (pending a question about versioning)

**Comment by slapresta** [comment]
#44 is merged so I'm closing this.
